docs: Improve doc about TLS (#1019)
redpanda-data/documentation: versioned_docs/version-22.3/deploy/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.mdx

Redpanda brokers must store their data on disk. As a result, each Pod that runs a Redpanda broker must have its own storage volume.

> For production, make sure to create and mount an **XFS file system** on any storage volumes that host the data directory of Redpanda (`/var/lib/redpanda/data`). XFS is a high-performance file system that provides important features for running Redpanda in production. NFS is not supported.

Reviewer: ```suggestion
For production, you must create and mount an **XFS file system** on any storage volumes that host the data directory of Redpanda (`/var/lib/redpanda/data`). XFS is a high-performance file system that provides important features for running Redpanda in production. NFS is not supported.
```
